std/net/curl.d

```diff
@@ -2879,7 +2879,7 @@ struct HTTP
     /** The headers read from a successful response.
      *
      */
-    @property string[string] responseHeaders()
+    @property string[string] responseHeaders() return
     {
         return p.headersIn;
```

Comment:
> This ties it to the lifetime of the struct.

The point I'm trying to make is that there's no reason that I can see to do that, and I don't understand why we can't let the AA outlive the struct, seeing as how it will always be managed by the GC and therefore doesn't present a safety risk.

Adding `return` here does not make this code any safer as far as I can tell.